func-consumption/check-pin-expiry/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "_ref": "sec, min, hour, day, month, dow",
```

Comment:
whats the `_ref` for?  is it an undocumented setting?

Reply:
It's a comment to remind the viewer what the format of the cron expression is.   As this is JSON commenting has to be a field.

Reply:
good idea 💡